spec/poros/quote_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+RSpec.describe Quote do
+  before :each do
+    quote_data = JSON.parse(file_fixture("single_quote_response.json").read, symbolize_names: true)
+    @quote = Quote.new(quote_data)
```

Comment:
Similar to the feature test you wrote, I really like that you opted to pass data directly to the `Quote` class rather than making a live API call for your test.